homes/development/editor.nix

```diff
@@ -0,0 +1,22 @@
+# SPDX-FileCopyrightText: 2025 FreshlyBakedCake
+#
+# SPDX-License-Identifier: MIT
+
+{ pkgs, ... }: {
+  programs.zed-editor = {
+    enable = true;
+    extensions = [
+      "catppuccin"
+      "catppuccin-icons"
+      "nix"
+    ];
+    userSettings = {
+      git.git_gutter = "hide";
+      minimap.show = "auto";
+      vim_mode = true;
+    };
+  };
+  home.packages = [
+    pkgs.nautilus
```

Comment:
Can this not be in development? Presumably we want a file manager for more programs than just zeditor